backend/src/govsg/services/govsg.service.ts

```diff
@@ -114,6 +115,21 @@ export function uploadCompleteOnPreview({
   }
 }
 
+const getLanguageCode = (language: string | undefined) => {
+  if (!language) {
+    return WhatsAppLanguages.english
+  }
+  const languageInLowerCase = language.toLowerCase()
```

Comment:
minor nit: but actually I think this is small enough to just be done inline on line 125

Reply:
I just want to be a bit clearer than making this function call feel like an afterthought. Would probably leave this in until it becomes a pain.